### DISCUSSION TITLE
Propose @mateusoliveira43 as the project maintainer

### DISCUSSION THREAD

**Discussion by mpryc** [dora-metrics/pelorus#805] [Polls]
I do propose @mateusoliveira43 to become project maintainer. He already created number of merged pull requests.

We do not have an established agreement nor guidelines on who becomes maintainer, so it's on discretionary basis.

Also having him as a maintainer will easy our conversations and proposals from his side to already existing PRs using [Allowing changes to a pull request branch created from a fork] (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Comment by mpryc** [comment]
Thank you for the votes. 3 up from the Pelorus contributors, so I will go ahead and make changes.

Some of them requires PR and it's already opened - PR #814.

This will be propagated to the relevant `OWNERS` files in the repo (no need to perform any action, this comment is FYI only to explain how this works):
   https://github.com/openshift/release

```
ci-operator/config/konveyor/pelorus/OWNERS
ci-operator/jobs/konveyor/pelorus/OWNERS
core-services/prow/02_config/konveyor/pelorus/OWNERS
```

A bit of background on the OWNERS file:
    https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md

**Comment by mateusoliveira43** [comment]
closing as done
